src/Compiler/AbstractIL/ilwritepdb.fs

```diff
@@ -324,7 +324,7 @@ let sortMethods info =
 
 let getRowCounts tableRowCounts =
     let builder = ImmutableArray.CreateBuilder<int>(tableRowCounts |> Array.length)
-    tableRowCounts |> Seq.iter (fun x -> builder.Add x)
+    tableRowCounts |> Seq.iter (builder.Add)
     builder.MoveToImmutable()
```

Comment:
Could this whole thing not just be `ImmutableArray.CreateRange tableRowCounts` or `tableRowCounts.ToImmutableArray()`?

Reply:
Probably, but I'm not too familiar with those constructs.

Reply:
https://source.dot.net/#System.Collections.Immutable/System/Collections/Immutable/ImmutableArray.cs,f22b27e8291026ba

Reply:
I think this could go for a different PR